package.yaml

```diff
@@ -0,0 +1,70 @@
+name:                servant-elm-street
+version:             0.1.0.0
+github:              "githubuser/servant-elm-street"
+license:             BSD3
+author:              "Author name here"
+maintainer:          "example@example.com"
+copyright:           "2021 Author name here"
```

Comment:
Fix these. Not sure what they should be though.

Reply:
@dtaskoff please verify this when you're here

Reply:
The copyright should state "Copyright 2021 Chaos Software OOD". The others fields can be filled as you like.

Reply:
done